ksql-rest-app/src/main/java/io/confluent/ksql/rest/server/StandaloneExecutor.java

```diff
@@ -90,12 +89,6 @@ public static StandaloneExecutor create(final Properties properties,
                                           final String queriesFile,
                                           final String installDir) {
     final KsqlConfig ksqlConfig = new KsqlConfig(properties);
-    Map<String, Object> streamsProperties = ksqlConfig.getKsqlStreamConfigProps();
```

Comment:
Was this always purely redundant since the application id was being set in the physical plan builder anyway?

Reply:
Yep, its not doing anything.